### ISSUE TITLE
Site Theme

### ISSUE THREAD

**Issue by TiberiusB** [bitmindco/ovn-space#12]
We looked around for the Pirat theme and people are not very happy with it... 
It would be nice to have a Theme that can add more flexibility to the website, and to allow almost everyone to create structure and content, under certain rules.

**Comment by dgrmunch** [comment]
Yes. Last summer I was looking for a theme which was attractive to present the OVN model to possible investors or interested contributors. I found the Pirate to be visual, responsive and appropriated for the structure I had in mind when I setup ovn.space. So I took it and adapted it with some basic customization, css, ... 
However, as always happens in WP, if you are not a PHP master (not my case, I am a Java-speaker mostly) it is very limited to the theme constraints and the widgets. I don't consider this as a problem though. When we started the ovn.space site everyone agreed in the need of creating a "hub" to "federate" content or to publish links to our respective sites. 
So ovn.space was not created as a portal itself. The wiki itself is more useful to serve as a more centralized place for "content". So I think that in ovn.space it is not gonna be very common to require more customization or updates, besides the posts and the content itself. If it is, we can discuss it in a thread or in the document (https://docs.google.com/document/d/1H735-yqHt_MC7TYQvG8inOjEwkQC30HslvQKXOKCOlg/edit?usp=sharing) and propose a new theme, sure! ;-)
As said, we don't want to centralize the OVN conversation in ovn.space. We wanted this to be kind of a window for outsiders, to help them to land into our ecosystem. Because of this, we added a link to a "Metamap" bellow the "members/team/nodes" section. The idea was put in the web some of first nodes of the ecosystem, but also invite the visitors to click on the link to the "Map" in order to see the whole ecosystem, which will be bigger than a landing page (hopefully ☺). 
Btw, we still need some work on the map, anyone is interested?

**Comment by dgrmunch** [comment]
The last point of my comment has its specific issue: https://github.com/bitmindco/ovn-space/issues/8
